src/wallet/walletutil.cpp

```diff
@@ -60,33 +60,69 @@ std::vector<fs::path> ListWalletDir()
     const size_t offset = wallet_dir.string().size() + 1;
     std::vector<fs::path> paths;
     boost::system::error_code ec;
+    fs::path sub_path = GetDataDir();
+
+    bool we_have_wallets_dir = (wallet_dir != sub_path);
+    const fs::path fblocks = GetBlocksDir();
```

Comment:
Instead of multiple variables, you can declare something like `const std::vector ignore_paths{ GetBlocksDir(), sub_path / "testnet3", ... }` and then compare against this.

Reply:
yup  :+1:  , btw strange how long those bare strings survived refactor, still no global #define BLOCKS_DIR or TESTNET_STR etc.